workflow/artifacts/s3/s3.go

```diff
@@ -29,6 +30,19 @@ type ArtifactDriver struct {
 	Context     context.Context
 }
 
+// s3TransientErrorCodes is a list of S3 error codes that are transient (retryable))))
```

Comment:
Remove the redundant `()))`

Reply:
Thanks! Removed those in 044196f88cb01a82961fcfe9620c64a73cbb8ceb.